README.md

```diff
@@ -61,6 +63,26 @@ The process of setting up is largely the same for different operating systems. H
 Go to  https://visualstudio.microsoft.com/visual-cpp-build-tools/ and click 'Download Build Tools'.
 Open the installer and make sure you are installing the package labeled *C++ build tools*, then follow the prompts.
     - **(If installing on CAC, you will need the admin password and must contact support@ionpath.com)**
+    - **Git - CAC:** We highly recommend installing git system-wide on the CAC, by downloading the installation utility [here](https://git-scm.com/downloads).
+       1. Under Standalone Installer, click the 64-bit Git for Windows Setup link to download the proper installer.
+       2. Run the Git setup `.exe` file. It should be version 2.37.1 or higher.
+       3. Click `Yes` to allow Git to make the necessary changes.
+       4. Click `Next` to accept the GNU License. 
+       5. Click `Next` to save Git in it's default location. ![alt text](templates/img/git_install/step_01.png "Select Destination Location")
```

Comment:
The guidance about which installers and versions to use is good to have, as well as how to check that it was installed correctly. However, since all of the other installation instructions are just "click next to accept the defaults", I think it can just be summarized that way. We don't need a screenshot of each step, or even to list the step, if there's nothing tricky for them to do. The "hard" part is making sure they've started the installation process correctly